source/Halibut.Tests/Support/TestCases/NetworkConditionTestCase.cs

```diff
@@ -17,15 +17,18 @@ public class NetworkConditionTestCase
 
         public static NetworkConditionTestCase NetworkConditionPerfect = 
             new (null, 
+                "Perfect", 
                 "Perfect");
 
         public static NetworkConditionTestCase NetworkCondition20MsLatency = 
             new ((i, logger) => PortForwarderBuilder.ForwardingToLocalPort(i, logger).WithSendDelay(TimeSpan.FromMilliseconds(20)).Build(), 
-                "20ms SendDelay");
+                "20ms SendDelay",
+                "20msLatency");
```

Comment:
Could we go even shorter with DLY for delay. Or even SD for Send Delay. I'm pretty whatevs, but just a thought 😁